src/components/Visualization.js

```diff
@@ -4,32 +4,33 @@ import LineChart from './LineChart'
 import CaVsNy from './CaVsNy'
 
 
-/* followed a tutorial for de-structuring and then rebuilding the object in Chart 
-was a good way to get familiar with API fetching and chartjs 
-but ran into a lot of errors so decided to implement a new 
+/* followed a tutorial for de-structuring and then rebuilding the object in Chart
+was a good way to get familiar with API fetching and chartjs
+but ran into a lot of errors so decided to implement a new
 method that made more sense to me (LineChart.js and CaVsNy.js) */
 
-function Visualization(){
+function Visualization() {
     const [stateData, setStateData] = useState([])
     const url = "https://covidtracking.com/api/v1/us/current.json"
 
-  useEffect(() => {
     async function getData() {
-      const res = await fetch(url);
-      const data = await res.json();
-      // must specify index, else returning undefined values 
-      setStateData(data[0]);
+        const res = await fetch(url);
+        const data = await res.json();
+        // must specify index, else returning undefined values
+        setStateData(data[0]);
     }
-    getData();
-  });
 
-    // calling all the charts and some styling! :-) 
-    return(
+    useEffect(() => {
```

Comment:
Attached a screencast of a bug here - why did this change fix it?
[rendering_bug.zip](https://github.com/cduong21/dataviz/files/4576120/rendering_bug.zip)